src/Payum/Core/Bridge/Propel2/Model/.gitignore

```diff
@@ -0,0 +1,2 @@
+Base/
+Map/
```

Comment:
the .gitignore should be in root folder, shoudn't it? If so the root folder is `src/Payum/Core/` in case you installed  `payum/core` and `/` in case of `payum/payum`. Should be added to both files.